modules/nf-core/mcquant/main.nf

```diff
@@ -0,0 +1,36 @@
+process MCQUANT {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "labsyspharm/quantification:1.5.4"
+
+    input:
+    tuple val(meta), path(image)
+    tuple val(meta2), path(mask)
+    tuple val(meta3), path(markerfile)
+
+    output:
+    tuple val(meta), path("*.csv"), emit: csv
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    // TODO: Replace fixed version number by mcquant command. PR to Mcquant needed!
```

Comment:
If the version isn't available, you should make it clear to users that they should update the version if they update the container. See [here](https://github.com/nf-core/modules/blob/master/modules/nf-core/wisecondorx/convert/main.nf) for an example